aspnetcore/security/authentication/accconfirm.md

```diff
@@ -79,7 +79,7 @@ Add the `[RequireHttps]` attribute to each controller. The `[RequireHttps]` attr
 
 ## Require email confirmation
 
-It's a best practice to confirm the email of a new user registration to verify they are not impersonating someone else (that is, they haven't registered with someone else's email). Suppose you had a discussion forum, you would want to prevent "bob@example.com" from registering as "joe@contoso.com". Without email confirmation, "joe@contoso.com" could get unwanted email from your app. Suppose Bob accidentally registered as  "bib@example.com" and hadn't noticed it, he wouldn't be able to use password recovery because the app doesn't have his correct email. Email confirmation provides only limited protection from bots and doesn't provide protection from determined spammers who have many working email aliases they can use to register.
+It's a best practice to confirm the email of a new user registration to verify they are not impersonating someone else (that is, they haven't registered with someone else's email). Suppose you had a discussion forum, you would want to prevent "some_user@example.com" from registering as "some_other_user@contoso.com". Without email confirmation, "some_other_user@contoso.com" could get unwanted email from your app. Suppose the user accidentally registered as  "some_ussr@example.com" and hadn't noticed the misspelling of "user," they wouldn't be able to use password recovery because the app doesn't have their correct email. Email confirmation provides only limited protection from bots and doesn't provide protection from determined spammers who have many working email aliases they can use to register.
```

Comment:
We could go with first initial and last name to avoid gender bias. How about ...

bsmith@example.com
jwilliams@contoso.com